app/create_file.py

```diff
@@ -1 +1,56 @@
-# write your code here
+from datetime import datetime
+from sys import argv
+from os import mkdir, chdir
+
+filename = ""
+directories = []
+
+if "-f" in argv:
+    index = 0
+
+    for _ in range(len(argv) - 1):
+        index += 1
+        if argv[_: _ + 1] == ["-f"]:
+            break
```

Comment:
Don't repeat yourself:
this code is duplicated many times; it can be placed in a separate function to avoid this. As example:
```
def get_argument_index(arg_flag, argv):
    arg_set = set(argv)
    if arg_flag in arg_set:
        return argv.index(arg_flag) + 1
    return None
```